api/tests/certification/session/unit/application/session-mass-import-route_test.js

```diff
@@ -0,0 +1,85 @@
+import { expect, HttpTestServer, sinon } from '../../../../test-helper.js';
+import { securityPreHandlers } from '../../../../../lib/application/security-pre-handlers.js';
+
+import * as moduleUnderTest from '../../../../../src/certification/session/application/session-mass-import-route.js';
+import { certificationCenterController } from '../../../../../lib/application/certification-centers/certification-center-controller.js';
+import FormData from 'form-data';
+import fs from 'fs';
+import { writeFile, stat, unlink } from 'fs/promises';
+import streamToPromise from 'stream-to-promise';
+
+import * as url from 'url';
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+
+describe('Unit | Router | session-mass-import-route', function () {
+  describe('POST /api/certification-centers/{certificationCenterId}/sessions/validate-for-mass-import', function () {
+    const testFilePath = `${__dirname}/testFile_temp.csv`;
+
+    let headers;
+    let payload;
+
+    beforeEach(async function () {
+      await writeFile(testFilePath, Buffer.alloc(0));
+      const form = new FormData();
+      const { size: knownLength } = await stat(testFilePath);
+      form.append('file', fs.createReadStream(testFilePath), { knownLength });
+
+      headers = form.getHeaders();
+      payload = await streamToPromise(form);
+    });
+
+    afterEach(async function () {
+      await unlink(testFilePath);
+    });
+
+    it('should exist', async function () {
+      // given
+      sinon.stub(securityPreHandlers, 'checkUserIsMemberOfCertificationCenter').callsFake((_, h) => h.response(true));
+      sinon
+        .stub(securityPreHandlers, 'checkCertificationCenterIsNotScoManagingStudents')
+        .callsFake((_request, h) => h.response(true));
+      sinon.stub(certificationCenterController, 'validateSessionsForMassImport').returns('ok');
+      const certificationCenterId = 123;
+      const httpTestServer = new HttpTestServer();
+      await httpTestServer.register(moduleUnderTest);
+
+      // when
+      const response = await httpTestServer.request(
+        'POST',
+        `/api/certification-centers/${certificationCenterId}/sessions/validate-for-mass-import`,
+        payload,
+        null,
+        headers,
+      );
+
+      // then
+      expect(response.statusCode).to.equal(200);
+    });
+
+    context('when user is member of a certification center from a sco organization managing student', function () {
+      it('should forbid access', async function () {
+        // given
+        sinon.stub(securityPreHandlers, 'checkUserIsMemberOfCertificationCenter').callsFake((_, h) => h.response(true));
+        sinon.stub(securityPreHandlers, 'checkCertificationCenterIsNotScoManagingStudents').callsFake((_request, h) => {
+          return Promise.resolve(h.response().code(403).takeover());
+        });
```

Comment:
Je suis un peu maniaque, pas besoin du return 😉
```suggestion
        sinon.stub(securityPreHandlers, 'checkCertificationCenterIsNotScoManagingStudents').callsFake((_request, h) => Promise.resolve(h.response().code(403).takeover()));
```